### PR TITLE
async function added to index route

### PR DESCRIPTION
**Pull request by HelenHTech**

### CODE REVIEW

**Review by Ma9sha — APPROVED** [review]
Understood that you need to have an await function to allow the programme to complete. in this scenario it allows all the data to be loaded from db and then displays it on console

File: app.js [hunk 1]
```diff
@@ -1,7 +1,12 @@
+const express = require('express')
+const app = express()
+const port = process.env.PORT || 8080;
```
Comment (melanierogan) [inline app.js:3]: Nice. Moving these lines up here means our express app knows what it needs before it starts to run

File: app.js [hunk 2]
```diff
@@ -16,13 +21,10 @@ db.once('open', function() {
   });
 });
 
+const xxx = db.collection('users')
```
Comment (melanierogan) [inline app.js]: we should probably change this const to something more descriptive, but we can do that next PR

File: app.js [hunk 3]
```diff
@@ -16,13 +21,10 @@ db.once('open', function() {
...
 
-const express = require('express')
-const app = express()
-const port = process.env.PORT || 8080;
-app.get('/', function (req, res) {
-  res.send('Hello World')
+app.get('/', async function (req, res) {
```
Comment (melanierogan) [inline app.js:26]: Lovely async await so that we can give the database time to retrieve what we asked for and show it on our route

**Review by melanierogan — APPROVED** [review]
Looking good to me, 👍

**Pull request merged**